dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/publish/servers/com.ibm.ws.ejbcontainer.security.jacc_fat.ejbjar/server.xml

```diff
@@ -20,6 +20,9 @@
 	</featureManager>
 	
 	<jaccTestProvider roleMappingFile="${server.config.dir}/resources/security/roleMapping.props" />
+	
+	<!--startEJBsAtAppStart is false by default. But in the case of checkpoint/restore, the default value is true. hence, setting false explicitly -->
+	<ejbContainer startEJBsAtAppStart="false"/>
```

Comment:
Is there an issue to track a fix for this such that setting this to `true` allows it to work for checkpoint.

Reply:
@tkburroughs, last time we talked, you suggested there could be a bug here that is unrleated to checkpoint.